### ISSUE TITLE
cannot close modal

### ISSUE THREAD

**Issue by qqdaiyu55** [chenjiahan/rodal#34]
```
<Rodal visible={this.state.isModalVisible} width={400} height={400} className="videoCard" onClose={this.closeNew} closeOnEsc={true}>
          <div>Content</div>
          <button onClick={this.closeNew}>close</button>
        </Rodal>
```
closeNew function
```
closeNew() {
    this.setState({isModalVisible: false});
  }
```

The modal cannot be closed when press button, with no error throwed. By the way, the mask is not shown (with default 'true'), and I cannot close the modal with 'ESC'.

**Comment by chenjiahan** [comment]
@qqdaiyu55 Can you provide a minimal demo of the problem via [codesandbox](https://codesandbox.io/s/mo2z9709oj)?

**Comment by qqdaiyu55** [comment]
It works well on codesandbox.. The screenshot in my environment is below, notice there's a weird blue border around the modal.

https://imgur.com/a/WuU0z

**Comment by chenjiahan** [comment]
Get, I will fix the outline style in next version

**Comment by chenjiahan** [comment]
fixed in v1.6.1
